Makefile

```diff
@@ -16,9 +16,9 @@ deps:
 
 clean:
 	./gradlew clean
-	docker rmi containersol/minimesos:latest
-	docker rmi containersol/mesos-hello-world-scheduler:latest
-	docker rmi containersol/mesos-hello-world-executor:latest
+	-docker rmi containersol/minimesos:latest
+	-docker rmi containersol/mesos-hello-world-scheduler:latest
+	-docker rmi containersol/mesos-hello-world-executor:latest
```

Comment:
Ignore non-zero exit codes for the command. If the image does not exist, this is fine because we wanted to delete it anyway.